imap/carddav_db.c

```diff
@@ -1120,12 +1130,18 @@ EXPORTED int carddav_store(struct mailbox *mailbox, struct vparse_card *vcard,
     time_to_iso8601(now, datestr, sizeof(datestr), 0);
     vparse_replace_entry(vcard, NULL, "REV", datestr);
 
+    /* Check size of vCard */
+    struct buf buf = BUF_INITIALIZER;
+    vparse_tobuf(vcard, &buf);
```

Comment:
For CardDAV, this check, including the VCARD dump, would be done twice, right?

Reply:
carddav_store() appears to only be called by jmap_contact().  We should probably try to find a way to consolidate that and http_carddav.c:store_resource()

Reply:
Thanks. Agree that we should consolidate, but also that this PR isn't the right place to do it.